settings.gradle

```
@@ -6,6 +6,7 @@
  */
 
 include ':yoga', ':yogacore', ':yoga-layout', ':yoga:proguard-annotations', ':libfb'
+include ':android:sample'
```

```suggestion
include ':yoga-layout:sample'
```
Or else it fails to sync with the message
```
ERROR: The modules ['android', 'yoga-layout'] point to the same directory in the file system.
Each module must have a unique path.
```
because when using relative project naming you implicitly declare an 'android' project

---

I don't see the error neither on windows nor on macOS.